### PR TITLE
Update: MegaManX816-bit 

### PR DESCRIPTION
**Pull request by Damian2809**
# New Update for Megamanx816-bit
## Game Information
- **Title**: Megamanx816-bit
- **URL**: https://github.com/AlyssonDaPaz/Mega-Man-X8-16-bit

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [X] AmberELEC
- [X] ArkOS
- [X] ROCKNIX
- [X] muOS

### Resolution Tests
Test all major resolutions:
- [x] 480x320
- [X] 640x480
- [X] 720x720 (RGB30)
- [X] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

**Comment by Cebion** [comment]
Why does the title say update port and the description says new port? The description field is there for a reason :)

**Comment by Cebion** [comment]
Json malformed, use https://jsonlint.com/ to check your json

File: ports/megamanx816-bit/port.json [hunk 1]
```diff
@@ -1,28 +1,28 @@
 {
-    "version": 3,
-    "name": "megamanx816-bit.zip",
-    "items": [
-        "Mega Man X8 16-bit.sh",
-        "megamanx816-bit"
+  "version": 3,
+  "name": "megaman816-bit.zip",
+  "items": [
+    MegaMan X8 16-bit.sh",
```
Comment (kloptops) [inline ports/megamanx816-bit/port.json:5]: This broken
```suggestion
    "MegaMan X8 16-bit.sh",
```

**Review by kloptops — CHANGES_REQUESTED** [review]

**Review by cdeletre — DISMISSED** [review]

File: ports/megamanx816-bit/megamanx816-bit/LICENSE.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+X8 16-bit License
```
Comment (JanTrueno) [inline ports/megamanx816-bit/megamanx816-bit/LICENSE.md:1]: We accept donations through the OpenCollective, doesnt that make the license invalid for PM?

**Review by JanTrueno — CHANGES_REQUESTED** [review]

File: ports/megamanx816-bit/megamanx816-bit/LICENSE.md (hunk 2 above)
Comment (Damian2809) [reply ports/megamanx816-bit/megamanx816-bit/LICENSE.md:1]: hmm fuck, i now wrote him on instagram and im hoping for an answer

**Pull request closed without merging**